testcases/open_posix_testsuite/conformance/interfaces/strchr/assertions.xml

```diff
@@ -0,0 +1,8 @@
+<assertions>
+  <assertion id="1" tag="ref:XBD7:64866:64870">
+    The function will locate the first occurrence of c (converted to a char)
+    in the string pointed to by s. The terminating NUL character is
```

Comment:
NUL character - should this be null byte?

Reply:
As per POSIX document (1003.1 2016 Edition) line number 64867, they specify \0 as 'NUL character'

Reply:
Oh wild... it must have been a recent change that I haven't kept up with in the past couple years. Thanks so much for the clarity!!